pandas_datareader/yahoo/daily.py

```diff
@@ -97,7 +98,9 @@ def yurl(symbol):
 
     def _get_params(self, symbol):
         unix_start = int(time.mktime(self.start.timetuple()))
-        unix_end = int(time.mktime(self.end.timetuple()))
+        day_end = self.end + datetime.timedelta(days=1) \
```

Comment:
@jreback As discussed above, the failure was intermittent, sometimes not returning the data for 12/31. I'M m not sure what value a test on ```r.end``` adds ... Thanks